src/capi/azext_capi/helpers/generic.py

```diff
@@ -18,3 +19,14 @@ def has_kind_prefix(inpt_str):
 def match_output(output, regexp=None):
     """Returns regex search result against given parameter"""
     return re.search(regexp, output) if regexp is not None else None
+
+
+def is_github_file_link(template):
+    """Returns true if is github file link, false for raw yaml files"""
+    if os.path.isfile(template):
+        return False
```

Comment:
```suggestion
        return True
```
I think the problem is that GitHub file link and local files are accepted but we are failing local files
Probability better to change method name to something like `is_github_file_link_or_local_file`.
Need to update `test_local_file` test as well

Reply:
I've updated this.